var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -132,27 +132,16 @@ def setup_run_environment(self, env):
             # After 0.99 podio installs its python bindings into a more standard place
             env.prepend_path("PYTHONPATH", self.prefix.python)
 
-        env.prepend_path("LD_LIBRARY_PATH", self.spec["podio"].libs.directories[0])
+        # Note: ROOT dependency automatically sets up ROOT environment vars
+
         if "+sio" in self.spec:
             # sio needs to be on LD_LIBRARY_PATH for ROOT to be able to
             # dynamicaly load the python bindings library
             env.prepend_path("LD_LIBRARY_PATH", self.spec["sio"].libs.directories[0])
 
-        # Frame header needs to be available for python bindings
-        env.prepend_path("ROOT_INCLUDE_PATH", self.prefix.include)
```

Comment:
Is this also done automatically for root dependent paths? I seem to remember having to add this manually, definitely after #13657.

Reply:
@tmadlener, yes this is done correctly as of https://github.com/spack/spack/pull/45109.

Reply:
Ah nice. I missed that.